src/prefect/automations.py

```diff
@@ -0,0 +1,165 @@
+from typing import Optional
+from uuid import UUID
+
+from pydantic import Field
+from typing_extensions import Self
+
+from prefect.client.utilities import get_or_create_client
+from prefect.events.schemas.automations import (
+    AutomationCore,
+    CompositeTrigger,
+    CompoundTrigger,
+    EventTrigger,
+    MetricTrigger,
+    MetricTriggerOperator,
+    MetricTriggerQuery,
+    Posture,
+    PrefectMetric,
+    ResourceSpecification,
+    ResourceTrigger,
+    SequenceTrigger,
+    Trigger,
+)
+from prefect.exceptions import PrefectHTTPStatusError
+from prefect.utilities.asyncutils import sync_compatible
+
+__all__ = [
+    "AutomationCore",
+    "EventTrigger",
+    "ResourceTrigger",
+    "Posture",
+    "Trigger",
+    "ResourceSpecification",
+    "MetricTriggerOperator",
+    "MetricTrigger",
+    "PrefectMetric",
+    "CompositeTrigger",
+    "SequenceTrigger",
+    "CompoundTrigger",
+    "MetricTriggerQuery",
+]
+
+
+class Automation(AutomationCore):
+    id: Optional[UUID] = Field(default=None, description="The ID of this automation")
+
+    @classmethod
+    @sync_compatible
+    async def create(cls: Self, automation: Self) -> Self:
```

Comment:
The pattern of 
```
Automation.create(Automation(...))
```
doesn't make sense to me (it's redundant).  I think a more natural spelling would be `Automation(**init_kwargs).create()` - an in-place API creation - or _maybe_ `Automation.create(**init_kwargs)` that both initializes and creates.  I have a preference for the former as I think it's more standard